src/sfloader/fluid_sffile.c

```diff
@@ -712,6 +721,43 @@ static int process_info(SFData *sf, int size)
             sf->romver.minor = ver;
             FLUID_LOG(FLUID_DBG, "ROM Version: %hu.%hu", sf->version.major, sf->version.minor);
         }
+        else if (chunk.id == DMOD_FCC)
+        {
+            /* Default modulators chunk
+             * https://github.com/spessasus/soundfont-proposals/blob/main/default_modulators.md
+             */
+            if (chunk.size % SF_MOD_SIZE != 0 || size == 0)
+            {
+                FLUID_LOG(FLUID_ERR, "DMOD chunk has invalid size (%d bytes)", chunk.size);
+                return FALSE;
+            }
+
+
+            // read the modulators sequentially
+            count = (int)(chunk.size / SF_MOD_SIZE) - 1; // minus the terminal record
```

Comment:
Here, you're casting to int, but `count` is declared `uint`, pls. decide for one.